distribution/docs/src/main/resources/_contents/introductioncontents.adoc

```diff
@@ -0,0 +1,44 @@
+Distributed Data Framework {branding} is a free and open-source common data layer that abstracts services and business logic from the underlying data structures to enable rapid integration of new data sources.
```

Comment:
I've been debating making them maven properties for a while now. I think this is a good reason to make that happen.